HOST lewis.lbl.gov
only_i 34
only_code None
force_amber_prep True
dry_run False
... 34 3kfq
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/kf/3kfq
3kfq.pdb 4phenix_3kfq.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/kf/pdb3kfq.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/kf/3kfq/3kfq.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/kf/pdb3kfq.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/kf/3kfq/3kfq.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 3kfq.pdb
===================================================

----------Chains
The following (original) chains have been found:
A
B
C
D

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
None

---------- Gaps (Renumbered Residues!)
gap of 3.135225 A between SER 24 and TRP 26
gap of 3.157132 A between SER 245 and TRP 247

---------- Mising heavy atom(s)

None
not prmtop_filename /net/lewis/scratch1/nigel/amber/hi_lo/kf/3kfq/4amber_3kfq.prmtop
not prmtop_filename /net/chevy/raid1/nigel/amber/hi_lo/kf/3kfq/4amber_3kfq.prmtop

	Running AmberPrep : 3kfq.pdb


==================================================
Running pdb4amber on 3kfq.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================

==================================================
Running elbow/antechamber for SCH 
==================================================

MoleculeClass :  C: 4  N: 1  O: 2  S: 2 (CHEMICAL COMPONENTS format)
	17 atoms
	16 bonds
	26 angles
	28 dihedrals
	0 rings (groups, rings, atoms) 0 0
	1 chirals : R 

~> antechamber -i 4antechamber_SCH.pdb -fi pdb -o SCH.mol2 -fo mol2       -nc -1 -m 1 -s 2 -pf y -c bcc -at gaff2 -ek "qm_theory='AM1',grms_tol=0.0005,scfconv=1.d-10,maxcyc=0,ndiis_attempts=700,"


~> parmchk2 -s 2 -i SCH.mol2 -f mol2 -o SCH.frcmod


==================================================
Preparing asu files and 4phenix_3kfq.pdb
==================================================

~> tleap -f 3kfq_asu_tleap_input_run

Checking output filenames
  file : 3kfq_asu.prmtop
Sorry:   Output file is empty : 3kfq_asu.prmtop
  Check contents of "leap.log"
